# Resize of image in the browser (s47)


## Rule Details

The file width and height should be adapted to the target size in the browser.

Examples of **incorrect** code for this rule:

```html

<img src="data.png" style="width:450px;height: 250px">

```

Examples of **correct** code for this rule:

```html

<img src="data.png" style="width:auto">

```