# Use a standard font

## Rule Details

The biggest disadvantage of using non-standard font is that it instantly affects the overall rendering speed of your pages.

Example of **incorrect** code for this rule:

```css
h4 { font-family: Code Pro; }
```

Alternative code for this rule:

```css
h4 { font-family: sans-serif }
```

